main.go

```diff
@@ -95,9 +95,9 @@ func init() {
 			Description: "Offer deals with fast retrieval",
 		},
 		{
-			Name:        "deal-start-window",
```

Comment:
I think this name was fine. We should fix the readme to say `--deal-start-window`.
Also, not changing names here won't invalidate existing configurations.